util/auth.js

```diff
@@ -0,0 +1,21 @@
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import secret from '../config/config';
+
+class Auth {
+  static hashPassword(password) {
+    bcrypt.hashSync(password, bcrypt.genSaltSync(10));
+  };
+
+  static compareHash(password, hashedPassword) {
+    bcrypt.compareSync(password, hashedPassword);
+  };
+
+  static genToken(email) {
```

Comment:
Are you just generating token using email? Don't you think you other parameters such as the userId?